python-package/tests/test_rgf_python.py

```diff
@@ -530,64 +292,55 @@ def test_params(self):
                                 learning_rate=-0.5,
                                 memory_policy='Generos',
                                 verbose=-1)
+        non_valid_params.update(add_non_valid_params)
         for key in non_valid_params:
-            reg.set_params(**valid_params)  # Reset to valid params
-            reg.set_params(**{key: non_valid_params[key]})  # Pick and set one non-valid parametr
-            self.assertRaises(ValueError, reg.fit, self.X_train, self.y_train)
+            est.set_params(**valid_params)  # Reset to valid params
+            est.set_params(**{key: non_valid_params[key]})  # Pick and set one non-valid parameter
+            self.assertRaises(ValueError, est.fit, self.X_train, self.y_train)
 
-    def test_attributes(self):
-        reg = self.regressor_class(**self.kwargs)
-        attributes = ('n_features_', 'fitted_', 'sl2_', 'min_samples_leaf_', 'n_iter_')
+    def test_attributes(self, add_attrs=None):
+        self.est = self.estimator_class(**self.kwargs)
+        add_attrs = [] if add_attrs is None else add_attrs
+        attributes = ['n_features_', 'fitted_', 'sl2_',
+                      'min_samples_leaf_', 'n_iter_', 'estimators_']
+        attributes.extend(add_attrs)
```

Comment:
```python
def test_attributes(self, add_attrs=()):
    attributes.extend(add_attrs)
```
is same?

Reply:
Yeah, thanks! We can get back to this in the future with neater solution.